lib/Platform.cs

```diff
@@ -190,8 +191,10 @@ static Platform()
                 using (var p = Process.Start(info))
                 {
                     var output = p.StandardOutput.ReadToEnd();
-                    // TODO: use Trace.WriteLine instead?
-                    System.Console.WriteLine("Output of sw_vers was: " + output);
+
+					using (var writer = new StreamWriter(p.StandardError))
```

Comment:
Something is wrong with the indentation here.

Reply:
Yes, this is because You are not using TABs instead of SPACEs.

Reply:
The rest of the file is formatted using spaces...